lib/child-writer/writer-proxy.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 resin.io
+ * Copyright 2017 resin.io
```

Comment:
Perhaps something to add to the Contributing guide? And maybe the contributing guide should also say that by submitting a PR to the Etcher repo, any 3rd-party authors are explicitly assigning their copyright over to resin.io?